### PR TITLE
Handle stage tokens

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

File: recipes/replace_vault_token.rb [hunk 1]
```diff
@@ -1,12 +1,27 @@
...
+  vault_host 'http://vault.service.stage-us-east-1.consul:8200'
+  data_bag_name 'vault'
+  data_bag_item_name 'tokens'
+  options(
+    ttl: '8670h',
+    policies: %w(default apps)
+  )
+  top_level_key 'stage'
+  min_remaining_ttl 3600
+  accessor_token data_bag_item('vault', 'tokens')['stage']['vault']['worker_token']
```
Comment (jeffbyrnes) [inline recipes/replace_vault_token.rb]: @eherot why the change to two separate, almost identical, resources, instead of the DRYer code previously in use?

Comment (eherot) [inline recipes/replace_vault_token.rb]: There was no DRY version of this. It previously only handled the prod environment. That said, this could be DRYer. Let me fix it.

Comment (jeffbyrnes) [inline recipes/replace_vault_token.rb]: The previous version looks like it could handle any environment, though it was only running on the prod-worker, so yeah, that makes sense.

**Comment by eherot** [comment]
@jeffbyrnes fixed.

**Comment by jeffbyrnes** [comment]
@eherot nice! Yeah, that’s way better. :shipit:
